### DISCUSSION TITLE
CORS

### DISCUSSION THREAD

**Discussion by emrahamet** [kubero-dev/ladder#42] [Q&A]
Will something be added to bypass CORS?

I am currently making additions to the code myself, and it needs to be recompiled with each new version.

**Comment by mms-gianni** [comment]
Sorry, not sure what you mean. 

CORS headers are not passed through the ladder proxy. If you still have some CORS errors, it is possibly because the URLs are not rewritten by the default rules to be passed through ladder. So might need to write some own rules to modify the URLs.

Every new version adds some new features. I try to keep backward compatibility. But as a young project, a lot of the code is moving and things might change.

**Reply by emrahamet** [reply]
I apologize for the poor English. I'm using ladder as a reverse proxy between the web browser and an API. The API I'm sending requests to is not responding to CORS requests, not responding to OPTIONS requests, and there is no Access-Control-Allow-Origin in the response. The proxy I'm currently using is corsproxy.io

**Reply by mms-gianni** [reply]
Ah ok, I think I'm getting it now. 

The ladder is currently limited to GET requests. I'm not so familiar with corsproxy.io but the ladder might not be a 1:1 replacement yet. 

But it might be an interesting feature in the future.

**Comment by deoxykev** [comment]
@emrahamet Next major release should support CORs; will ping you when it's ready.
